src/modules/ipfix/IpfixPrinter.cpp

```diff
@@ -323,6 +322,20 @@ void PrintHelpers::printFrontPayload(InformationElement::IeInfo type, IpfixRecor
 	fprintf(fh, "'");
 }
 
+/**
+ * Converts an u64 value to ntp64
+ * Attention: little endianess is assumed
```

Comment:
We should fix this to make it work for little and big endian.

Are `flowStartNanoseconds` and `flowEndNanoseconds` the only IEs where we assume little endian?

Reply:
I'll check it and come back with an answer.
The reason I did it like this is because the argument passed when calling this new function is obtained from ntohll, which already assumes little endianess.
Regardless, I agree that they have to be fixed.